addrecord.php

```diff
@@ -0,0 +1,10 @@
+<?php
+session_start();
+
+require_once 'vendor/autoload.php';
+// $displayFormSucces = false;
+$_SESSION['formSuccess'] = 'Record added to collection :)';
```

Comment:
Rather than using session here, it would be better to instead use a query parameter when you use header to redirect back to index.

Using session feels like overkill